[2024-05-16T11:31:46.350+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:31:46.368+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 manual__2024-05-16T11:27:47.297038+00:00 [queued]>
[2024-05-16T11:31:46.373+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 manual__2024-05-16T11:27:47.297038+00:00 [queued]>
[2024-05-16T11:31:46.373+0000] {taskinstance.py:2303} INFO - Starting attempt 5 of 10
[2024-05-16T11:31:46.380+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-16 11:27:47.297038+00:00
[2024-05-16T11:31:46.385+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1088) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:31:46.386+0000] {standard_task_runner.py:63} INFO - Started process 1089 to run task
[2024-05-16T11:31:46.385+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'manual__2024-05-16T11:27:47.297038+00:00', '--job-id', '534', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpp4f7sqro']
[2024-05-16T11:31:46.386+0000] {standard_task_runner.py:91} INFO - Job 534: Subtask import_table2
[2024-05-16T11:31:46.413+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 manual__2024-05-16T11:27:47.297038+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:31:46.462+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-16T11:27:47.297038+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-16T11:27:47.297038+00:00'
[2024-05-16T11:31:46.462+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:31:46.468+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:31:46.469+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:31:46.474+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:31:46.531+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:31:46.531+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:31:46.544+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240516T112747, start_date=20240516T113146, end_date=20240516T113146
[2024-05-16T11:31:46.572+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:31:46.583+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:31:46.584+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
